pandas/core/tools/datetimes.py

```diff
@@ -550,7 +550,8 @@ def _adjust_to_origin(arg, origin, unit):
 
 
 @overload
-def to_datetime(
+# error: Overloaded function signatures 1 and 3 overlap with incompatible return types
```

Comment:
Hmm do you know why we are getting this error? Seems like an issue with the annotation

Reply:
since np.ndarray resolves to Any, this will need ignores if merging now b4 the numpy types are available. will close and include in #36092 instead.